sympy/logic/boolalg.py

```diff
@@ -640,7 +638,7 @@ def _apply_patternbased_simplification(self, rv, patterns, measure,
                 results = list(reversed(sorted(results,
                                                key=lambda pair: pair[0])))
                 # Replace the one providing most simplification
-                cost, replacement = results[0]
+                replacement = results[0][1]
```

Comment:
cost not used.

Reply:
I often use this pattern to make the code more readable and also to force an error to be raises if `results[0]` has the wrong length:
```python
In [1]: cost, replacement = [1]
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1-98a393f56ecc> in <module>
----> 1 cost, replacement = [1]

ValueError: not enough values to unpack (expected 2, got 1)
```